### DISCUSSION TITLE
Why does the vite config skip component scss files?

### DISCUSSION THREAD

**Discussion by andrewatduckpin** [flyntwp/flynt#516] [General]
I noticed that when I make changes to SCSS files within components, I don't get compile/reload. Looking at the vite config, we're watching `./Components/**/*.{php,twig}`

I updated that to include the SCSS files `{php,twig,scss}` which works, but I'm curious why it wasn't watching for some of the most changed files in the build already? Maybe I'm missing something!

**Comment by domtra** [comment]
@andrewatduckpin Could it be that you use docker for development? If so, this is a vite issue with hmr and docker. Maybe see https://github.com/vitejs/vite/issues/4116 for some help. 
Or what environment are you developing in? we do not seem to experience this issue in our workflow.
